backend/EkoFunkcje/EkoFunkcje/Features/Comments/AddCommentFunction.cs

```diff
@@ -37,12 +34,13 @@ public static class AddCommentFunction
                 string json = JsonConvert.SerializeObject(errorList);
                 return new BadRequestObjectResult(json);
             }
-            // I'm not sure if the best option is to create an extra table for comments only -> maybe extra column in Interventions will be fine
-            CommentEntity commentEntity = new CommentEntity(commentDto.InterventionId);
+
+            CommentEntity commentEntity = new CommentEntity();
```

Comment:
Wydaje mi się, że jeśli pola InterventionId oraz Comment są wymagane przez CommentEntity, to dobrą praktyką byłoby wymusić ich wstrzykiwanie przez konstruktor albo stworzenie statycznej factory method na klasie CommentEntity, która by wymagała tych parametrów. Samo CommentEntity miałoby wtedy tylko prywatny konstruktor wołany przez tą statyczną metodę.

Reply:
Nie mam w ogole takiego poczucia ze te komentarze powinny byc trzymane w osobnej tabeli. Jak dla mnie to co najwyzej dodatkowe pole przy interwencji z jsonsem jakby trzeba bylo wyroznic te komentarze jako rozne. Chociaz w sumie bardziej to rozumiem jako taka notatka

Reply:
Zmienione według sugestii Darka